CHANGELOG

```diff
@@ -122,15 +122,14 @@ v 7.9.4
   - Fixed issue where only 25 commits would load in file listings
   - Fix LDAP identities  after config update
 
-v 7.9.3
-  - Contains no changes
+v 7.9.3 (GitLab-EE)
   - Add icons to Add dropdown items.
   - Allow admin to create public deploy keys that are accessible to any project.
   - Warn when gitlab-shell version doesn't match requirement.
   - Skip email confirmation when set by admin or via LDAP.
   - Only allow users to reference groups, projects, issues, MRs, commits they have access to.
 
-v 7.9.3
+v 7.9.3 (GitLab-CE, Gitlab-CI)
```

Comment:
@maxlazio .. I have rebased, sorry for the confusion. Anyway, the changelog is quite confusing, I believe breaking the rules is quite necessary here or else the EE change should be ommitted to avoid the confusion. But the later would not suffice for enterprise users so I believe that is necessary